app/services/IdService.scala

```diff
@@ -95,7 +91,7 @@ class IdService @Inject()(val dateHelper: DateHelper,
           case true => (Some(true), Some(true))
           case false => (Some(false), Some(false))
         }
-      case ids => Future(ids._1, ids._2)
+      case ids => Future((ids._1, ids._2))
```

Comment:
Instead of creating a new asynchronous computation and run it with Future.apply, please, take the shortcut to an already completed Future with Future.sucessful